src/plugins/kibana_usage_collection/server/collectors/core/core_usage_collector.ts

```diff
@@ -0,0 +1,144 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import {
+  CoreUsageData,
+  CoreUsageDataStart,
+  // eslint-disable-next-line @kbn/eslint/no-restricted-paths
+} from '../../../../../core/server/core_usage_data/types';
```

Comment:
Should we just export these? I think with TS project references we'll have to anyways right?